### PR TITLE
unpkg in scripting examples

### PR DESCRIPTION
**Pull request by padawannn**

### CODE REVIEW

File: examples/carto/scripting/bq-tiler-layer.html [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <html>
   <head>
     <!-- deck.gl standalone bundle -->
-    <script src="http://127.0.0.1:8080/modules/core/dist.min.js"></script>
-    <script src="http://127.0.0.1:8080/modules/layers/dist.min.js"></script>
+    <script src="https://unpkg.com/deck.gl@^8.1.0/dist.min.js"></script>
```
Comment (alasarr) [inline examples/carto/scripting/bq-tiler-layer.html:4]: Let's better start with the packages instead of a whole import

**Review by alasarr — CHANGES_REQUESTED** [review]

File: examples/carto/scripting/bq-tiler-layer.html (hunk 1 above)
Comment (VictorVelarde) [reply examples/carto/scripting/bq-tiler-layer.html:4]: I would keep it, but updating it to latest version, not 8.1 but 8.2.0-beta.3

Comment (padawannn) [reply examples/carto/scripting/bq-tiler-layer.html:4]: In other scripting examples, they use the whole import

Comment (padawannn) [reply examples/carto/scripting/bq-tiler-layer.html:4]: TODO added

**Review by alasarr — APPROVED** [review]

**Pull request merged**